typesafety/test_maybe/test_maybe_type.yml

```diff
@@ -142,9 +142,9 @@
   disable_cache: false
   main: |
     from returns.maybe import Maybe
-    from typing import NoReturn
```

Comment:
This change is dangerous, because not python versions have `typing` with `Never`, please use `typing_extensions`.

Reply:
done